### DISCUSSION TITLE
Videos extra CSS

### DISCUSSION THREAD

**Discussion by ghost** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#235] [Q&A]
Buenas noches compañeros, una duda donde se encuentran los videos de explicación de css?

**Comment by doguedogue** [comment]
aquí mira https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a

**Reply by JuanSalvador73** [reply]
Mi Gracias :D
